textproc/grc/Portfile

```diff
@@ -21,28 +21,31 @@ long_description    Generic Colouriser makes text files or outputs of different
                     It provides the grc and grcat Python scripts for filtering \
                     text output and colourising it.
 
-checksums           rmd160  f494d0850098fc81ca26d623ed9f01cbe0d44b3c \
-                    sha256  7c76fff1b0dc70a27b07fb3f0f7007518afa07f6804019037957a922a2ac98ec \
-                    size    45192
+checksums           rmd160  06578c83ffae331825fc1b5fe8ac1b5279f1f44a \
+                    sha256  1468c50237fc260840384cc2063f3031689e95e5c6dc22533aa5d25a3d17f2f9 \
+                    size    49236
 
-python.versions     27 35 36
+python.versions     27 39 310
```

Comment:
this line doesn't do anything if the portname doesn't start with 'py-' so it should be removed. I personally don't see the need here for variants - why don't we just pick one Python version and use that (in this case 3.9 as that is currently MacPorts' default).

If you agree with that, this line should just become `python.default_version 39` and you remove the whole caboodle with the variants and notes.

Reply:
@reneeotten thank you very much for your feedback! 
i wasn't aware of what you explained to me in the comment, amazing. and duly noted.
i amended the PR as you suggested and changing the commit message accordly.

ps i just left he note I added since it is from the upstream and made me think a macports user using python27 must be aware they could be some weird/wrong beaviours using py27 (not to mention it's likely py27 supposed is deemed to be dropped sooner than later. please let me know if that make sense and it's ok for you.

cheers,
--
ferdy

Reply:
@reneeotten thank you very much for your feedback!

i wasn't aware of what you explained to me in the comment, amazing. and duly noted.
i amended the PR as you suggested and changed the commit message accordingly.

ps, i just left the note I added since it is from the upstream and made me think a MacPorts user using python27 must be aware they could be some weird/wrong behaviours using py27 (not to mention it's likely py27 supposed is deemed to be dropped sooner than later. please let me know if that makes sense and it's ok for you.

cheers,